app/forms/schools/add_participants/wizard_steps/who_to_add_participant_checks.rb

```diff
@@ -58,11 +58,7 @@ def transfer_step_for_type
         end
 
         def cohort_checks
-          if wizard.needs_to_confirm_start_term?
-            # we're not sure at this point which cohort is needed
-            # so allow to proceed at this point
-            :none
-          elsif !wizard.registration_open_for_participant_cohort?
+          if !wizard.registration_open_for_participant_cohort?
```

Comment:
I would not delete this

Reply:
I think we still need the logic behind `wizard.needs_to_confirm_start_term?`

Reply:
The only remaining logic in this method is `!Cohort.within_automatic_assignment_period?` given we no longer apply the clauses `(mentor_participant? || induction_start_date.blank?)` so I've re-added this as 

```ruby
if !Cohort.within_automatic_assignment_period?
  :none
```